README.md

```diff
@@ -168,21 +93,11 @@ the following while in your virtual enviroment:
 Once the above is done you will have autoformatting and pep8 compliance built
```

Comment:
all this pre commit stuff is covered in the shared readmes - please replace with links